### PR TITLE
ETFE-2652 CAM-ITM12: wine-operations-choice

### PR DESCRIPTION
**Pull request by grahampaulcook**

### CODE REVIEW

**Comment by grahampaulcook** [comment]
Updating conflicts

File: app/views/sections/items/ItemWineOperationsChoiceView.scala.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+
+@(form: Form[_], action: Call, values: Seq[WineOperations])(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("itemWineOperationsChoice.title")), maybeShowActiveTrader = maybeShowActiveTrader(request)) {
+
+    @formHelper(action = action, 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
```
Comment (stephenkward) [inline app/views/sections/items/ItemWineOperationsChoiceView.scala.html:41]: error message does not show above the checkboxes, it only shows at top of page.
image

File: conf/messages [hunk 2]
```diff
@@ -1259,8 +1259,17 @@ itemImportedWineChoice.error.required = Select yes if the wine was imported from
...
+
+itemWineOperationsChoice.title = Has the wine undergone any operations?
+itemWineOperationsChoice.heading = Has the wine undergone any operations?
+itemWineOperationsChoice.hint = Select all that apply.
+itemWineOperationsChoice.checkYourAnswersLabel = Wine operations
+itemWineOperationsChoice.error.required = Select a wine operation, or select 'No, I am not aware that the wine has undergone any operations'
+itemWineOperationsChoice.error.invalid = Select a wine operation, or select 'No, I am not aware that the wine has undergone any operations'
+itemWineOperationsChoice.error.exclusive = Select a wine operation, or select 'No, I am not aware that the wine has undergone any operations'
+itemWineOperationsChoice.change.hidden = wine operations
+itemWineOperationsChoice.0 = No, I am not aware that the wine has undergone any operations
```
Comment (stephenkward) [inline conf/messages:1275]: don't think this message is being used?

**Review by stephenkward — CHANGES_REQUESTED** [review]

File: app/views/sections/items/ItemWineOperationsChoiceView.scala.html (hunk 1 above)
Comment (grahampaulcook) [reply app/views/sections/items/ItemWineOperationsChoiceView.scala.html]: Updated https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/199/commits/128bbca894d5c7506a7996c15fe00b91da42f247

**Review by stephenkward — APPROVED** [review]

**Pull request merged**